src/rp2_common/pico_cyw43_driver/cyw43_bus_pio_spi.c

```diff
@@ -84,6 +84,16 @@ typedef struct {
 
 static bus_data_t bus_data_instance;
 
+int cyw43_spi_clock_div_int = CLOCK_DIV;
+int cyw43_spi_clock_div_frac = CLOCK_DIV_MINOR;
+
+// Allow the user to adjust the SPI clock speed 
+// prior to initialization.
+void cyw43_spi_set_clock_divisor(int clock_div_int, int clock_div_frac) {
```

Comment:
I wonder if that's the right name for the function. Maybe cyw43_set_pio_clock_divisor?

Reply:
Good call, done